src/openms/source/ANALYSIS/OPENSWATH/OpenSwathWorkflow.cpp

```diff
@@ -523,7 +523,6 @@ namespace OpenMS
           int outer_thread_nr = omp_get_thread_num();
           omp_set_num_threads(std::max(1, total_nr_threads / threads_outer_loop_) );
```

Comment:
sure that this does not result in any error if we have no OpenMP?

Reply:
and it seems I've overlooked a function :-/